tests/ft_sh.vader

```diff
@@ -12,3 +12,6 @@ Execute (shellcheck):
   set ft=sh
   norm i#!/usr/bin/env bash
   AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc']
+
+  norm ggcG# shellcheck shell=bash
```

Comment:
I would prefer to clean up later so as not to have this pull request include unrelated changes.